packages/cactus-core-api/src/main/typescript/plugin/ledger-connector/i-socket-api-client.ts

```diff
@@ -8,15 +8,19 @@ import type { Observable } from "rxjs";
  */
 export interface ISocketApiClient<BlockType> {
   sendAsyncRequest?(
-    contract: Record<string, unknown>,
-    method: Record<string, unknown>,
     args: any,
+    method?: Record<string, unknown>,
```

Comment:
@StepniowskiP Seems like a breaking change. Could you please refactor it to be backwards compatible with `1.0.0`?

Reply:
Sure, I have refactored method parameter to one before my changes

Reply:
@StepniowskiP Is it just me or the parameters are still out of order (e.g. not backwards compatible)?
This is the diff I'm looking at:

![image](https://user-images.githubusercontent.com/1583231/170373446-2afa4fef-ce7b-49d1-b2f1-a4e1ca32f0af.png)